Source/BSN.Commons.PresentationInfrastructure/Responses/GenericResponseBaseWithPagination.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace BSN.Commons.Responses
+{
+    public class GenericResponseBaseWithPagination<T> : GenericResponseBase<T> where T : class
```

Comment:
Please add comments for this class

Reply:
I resolved it

Reply:
Please describe the main responsibility of this class

The cause of existence of this class must be describe carefully

thanks

Reply:
The user must be know from your comments, that when he must use this class.

Reply:
Good ;)